ts-tests/test-api.tsx

```diff
@@ -26,6 +26,14 @@ const testStyledCss = () => {
         background: black;
     `;
 
+    const EmptyPropsText = styled('p')`
```

Comment:
These components would have previously failed the `test-ts` script.

Reply:
Really loving this!